CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ project(
 if(NOT DEFINED ENV{FOAM_SRC})
   message(FATAL_ERROR "You must source OpenFOAM before building FOAM_ADAPTER")
 endif()
-if(NOT ENV{FOAM_API} STREQUAL "2406")
+if(NOT "$ENV{FOAM_API}" STREQUAL "2406")
```

Comment:
Could you add your solution
```
if(NOT "$ENV{FOAM_API}" GREATER "2312")
  message(WARNING "OpenFOAM version < 2406")
endif()
```

Reply:
Done. (Using integer comparison.)

Reply:
I'll merge it once the pipeline is done.

Reply:
Thanks!